test/signing/sdkSign.ts

```diff
@@ -0,0 +1,188 @@
+    }
+    const rfqOrderSig = await signingHelper.signRFQOrder(rfqOrder, {
+        type: SignatureType.EIP712,
+        signer: signer,
+        verifyingContract: rfqPayloadJson.RFQ,
+    })
+    rfqPayloadJson["expectedOrderSig"] = rfqOrderSig
+
+    const rfqFill: RFQFill = { ...rfqOrder, receiverAddr: rfqPayloadJson.receiverAddr }
+    const rfqFillSig = await signingHelper.signRFQFillOrder(rfqFill, {
+        type: SignatureType.EIP712,
+        signer: signer,
+        verifyingContract: rfqPayloadJson.RFQ,
+    })
+    rfqPayloadJson["expectedFillSig"] = rfqFillSig
+
+    fs.writeFileSync(RFQPath, JSON.stringify(rfqPayloadJson, null, 2))
+}
+
+async function signLimitOrderOrderAndFill(signer: Wallet) {
+    const limitOrder: LimitOrder = {
+        makerToken: limitOrderPayloadJson.makerToken,
+        takerToken: limitOrderPayloadJson.takerToken,
+        makerTokenAmount: limitOrderPayloadJson.makerTokenAmount,
+        takerTokenAmount: limitOrderPayloadJson.takerTokenAmount,
+        maker: limitOrderPayloadJson.maker,
+        taker: limitOrderPayloadJson.taker,
+        salt: limitOrderPayloadJson.salt,
+        expiry: limitOrderPayloadJson.expiry,
+    }
+    const limitOrderSig = await signingHelper.signLimitOrder(limitOrder, {
+        type: SignatureType.EIP712,
+        signer: signer,
+        verifyingContract: limitOrderPayloadJson.LimitOrder,
+    })
+    limitOrderPayloadJson["expectedOrderSig"] = limitOrderSig
+    const orderHash = await signingHelper.getLimitOrderEIP712Digest(limitOrder, {
+        chainId: CHAIN_ID,
+        verifyingContract: limitOrderPayloadJson.LimitOrder,
+    })
+
+    const limitOrderFill: LimitOrderFill = {
+        orderHash: orderHash,
+        taker: limitOrder.taker,
+        recipient: limitOrderPayloadJson.recipient,
+        takerTokenAmount: limitOrder.takerTokenAmount,
+        takerSalt: limitOrder.salt,
+        expiry: limitOrder.expiry,
+    }
+    const limitOrderFillSig = await signingHelper.signLimitOrderFill(limitOrderFill, {
+        type: SignatureType.EIP712,
+        signer: signer,
+        verifyingContract: limitOrderPayloadJson.LimitOrder,
+    })
+    limitOrderPayloadJson["expectedFillSig"] = limitOrderFillSig
+
+    const limitOrderAllowFill: LimitOrderAllowFill = {
+        orderHash: orderHash,
+        executor: limitOrder.taker,
+        fillAmount: limitOrder.takerTokenAmount,
+        salt: limitOrder.salt,
+        expiry: limitOrder.expiry,
+    }
+    const limitOrderAllowFillSig = await signingHelper.signLimitOrderAllowFill(
+        limitOrderAllowFill,
+        {
+            type: SignatureType.EIP712,
+            signer: signer,
+            verifyingContract: limitOrderPayloadJson.LimitOrder,
+        },
+    )
+    limitOrderPayloadJson["expectedAllowFillSig"] = limitOrderAllowFillSig
+
+    fs.writeFileSync(LimitOrderPath, JSON.stringify(limitOrderPayloadJson, null, 2))
+}
+
+async function signL2Deposit(signer: Wallet) {
```

Comment:
Note: we need a release from `tokenlon-contract-lib-js` which includes the L2Deposit signing helpers. cc @pilagod

Reply:
@NIC619 just upgrade the library to v0.0.2 💪

https://www.npmjs.com/package/@tokenlon/contracts-lib